.github/workflows/test.yml

```
@@ -0,0 +1,92 @@
+        cache: false
+        
+    - name: Download dependencies
+      shell: pwsh
+      run: |
+        Write-Host "Current directory: $(Get-Location)"
+        Write-Host "Rust version: $(rustc --version)"
+        Write-Host "Cargo version: $(cargo --version)"
+        Write-Host "Files in root:"
+        Get-ChildItem
+        Write-Host "Building project:"
+        try { 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)